packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -755,4 +776,124 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /internal/patient/ops/coverage-assessment/validate
+ *
+ */
+router.post(
+  "/ops/coverage-assessment/validate",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
```

Comment:
What's the thought process for having it in diff endpoints? It seems we're doing validation here that we either already have on the zod schema or should be doing on the `POST /ops/coverage-assessment` endpoint as well.

I suggest we keep in one endpoint and add a `dryrun: boolean` parameter that we could pass to the endpoint just to see if it would work if we call it for real.

Reply:
I can combine with the dryrun param -- good call 👍

Reply:
Done.

Reply:
I see you combined the endpoints, which is great! I'd just try to have the dryrun to include the most of the logic possible. In the case of `/coverage-assessment`, we could execute all the code until `createCoverageAssessments`, right? It has some additional checks that are likely useful for someone testing the endpoint.

Reply:
Good call -- moved the return.